internal/pkg/skuba/kubernetes/versions_test.go

```diff
@@ -214,6 +214,14 @@ func TestAddonVersionForClusterVersion(t *testing.T) {
 			addon:          "not-exist",
 			expectErr:      true,
 		},
+		addonTestData{
+			name:           "Cilium 1.7 for 1.18",
+			clusterVersion: version.MustParseSemantic("1.18.6"),
+			addon:          "cilium",
+			expectVersion:  "1.7.6",
```

Comment:
That seems very fragile, shouldn't we just have a different test function, testing loading cluster versions, and expecting versions 1.7+ nstead?

What did you want to achieve in here?